src/components/Staking/StakingStatsBox.tsx

```diff
@@ -1,78 +1,78 @@
-import styled from "@emotion/styled"
+import React, { useState, useEffect, ReactNode } from "react"
 import { useIntl } from "react-intl"
-import { Spinner } from "@chakra-ui/react"
+import { Code, Flex, Icon, Spinner } from "@chakra-ui/react"
 // Import components
 import Translation from "../Translation"
 import Tooltip from "../Tooltip"
 import Link from "../Link"
-import Icon from "../Icon"
 // Import utilities
 import { Lang } from "../../utils/languages"
 import { getData } from "../../utils/cache"
 import { getLocaleForNumberFormat } from "../../utils/translations"
+import { MdInfoOutline } from "react-icons/md"
 
 // Constants
 const NA_ERROR = "n/a"
 const ZERO = "0"
 const MAX_EFFECTIVE_BALANCE = 32
 
-// Styled components
-const Container = styled.div`
-  display: flex;
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    flex-direction: column;
-  }
-`
-
-const Cell = styled.div`
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding: 1rem 2rem;
-  border-left: 1px solid ${({ theme }) => theme.colors.preBorder};
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    border-left: none;
-    border-top: 1px solid #33333355;
-  }
-  &:first-child {
-    border-left: none;
-    border-top: none;
-  }
-`
-
-const Value = styled.code`
-  font-weight: 700;
-  font-size: 2rem;
-  background: none;
-  display: flex;
-  align-items: center;
-  text-align: center;
-  text-transform: uppercase;
-  color: ${({ theme }) => theme.colors.primary};
-`
+const Cell: React.FC<{ children: ReactNode }> = ({ children }) => {
+  return (
+    <Flex
+      direction="column"
+      alignItems="center"
+      py={4}
+      px={8}
+      borderLeft={{ base: "none", md: "1px solid" }}
+      borderLeftColor={{ md: "preBorder" }}
+      borderTop={{ base: "1px solid #33333355", md: "none" }}
+      sx={{
+        "&:first-child": {
+          borderLeft: "none",
+          borderTop: "none",
+        },
+      }}
+    >
+      {children}
+    </Flex>
+  )
+}
 
-const Label = styled.div`
-  display: flex;
-  flex-wrap: nowrap;
-  align-items: center;
-  text-transform: uppercase;
-  font-size: 0.875rem;
-  margin-top: 0.5rem;
-`
+const Value: React.FC<{ children: ReactNode; title: string }> = ({
+  children,
+  title,
+}) => {
+  return (
+    <Code
+      title={title}
+      fontWeight="bold"
+      fontSize="2rem"
+      background="none"
+      display="flex"
+      alignItems="center"
+      textAlign="center"
+      textTransform="uppercase"
+      color="primary"
+    >
+      {children}
+    </Code>
+  )
+}
 
-const StyledIcon = styled(Icon)`
-  fill: ${({ theme }) => theme.colors.text};
-  margin-inline-start: 0.5rem;
-  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
-  }
-  &:hover,
-  &:active,
-  &:focus {
-    fill: ${({ theme }) => theme.colors.primary};
-  }
-`
+const Label: React.FC<{ children: ReactNode }> = ({ children }) => {
+  return (
+    <Flex
+      wrap="nowrap"
```

Comment:
Nit pick police again with the original code! 👮🏼‍♂️ 😅 

`wrap` is ineffective here and does not appear that it was trying to solve anything.